server/docs/swagger/report.yaml

```diff
@@ -0,0 +1,141 @@
+paths:
+  /articles/{slug}/reports/:
+    post:
+      tags:
+        - Report
+      summary: create a new article report
+      produces:
+      - application/json
+      parameters:
+      - name: authorization
+        in: header
+        description: authorization token
+        type: string
+      - name: slug
+        in: path
+        description: path parameter for the article slug
+        required: true
```

Comment:
remove the `required: true` to allow error message from the server sent to swagger